### ISSUE TITLE
Remove LiveCache/WorkingCache

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#10947]
We need to remove the Live and Working cache as they are no longer required.

Instead we follow the pattern of getting the identifier first then the appropriate live/working version from the version_info tables, e.g. to get the working version:
```
Identifier id = APILocator.getIdentifierAPI().find(host, uri);
ContentletVersionInfo cinfo = APILocator.getVersionableAPI().getContentletVersionInfo( id.getId(), languageId );
Contentlet c = APILocator.getContentletAPI().find( cinfo.getWorkingInode(), APILocator.getUserAPI().getSystemUser(), false );

```

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/11101

**Comment by bryanboza** [comment]
Fixed, tested on master-4.1 // Postgres // FF .... Working around containers and everything works normally
